app/assets/stylesheets/topbar.scss

```diff
@@ -5,6 +5,12 @@
 
   padding: 0px;
 
+  a {
+    &:hover {
+      color: lighten( $brick, 10% );
```

Comment:
Expected 0 spaces after `(` instead of ` `<br>Expected 0 spaces before `)` instead of ` `